database/PatientCheckInDBScript.sql

```diff
@@ -0,0 +1,143 @@
+create database PatientCheckIn
```

Comment:
sql creation scripts for each table should be in separate .sql file. There are 3 files which are placeholders to put the script.